web-ui/src/context/selectors.js

```diff
@@ -232,18 +232,26 @@ export const selectPDLCheckinMap = createSelector(selectCheckins, (checkins) =>
 export const selectSupervisors = createSelector(
   selectCurrentMembers,
   selectProfileMap,
-  (currentMembers, memberProfileMap) => currentMembers?.reduce((supervisors, currentMember) => {
-    let supervisorId = currentMember.supervisorid;
+  (currentMembers, memberProfileMap) => {
+    const filteredMembers = currentMembers?.filter(member => member.supervisorid);
 
-    const inSupervisors = supervisors.find(
-      (supervisor) => supervisorId === supervisor?.id
-    )
-
-      if (!inSupervisors){
-        supervisors.push(memberProfileMap[supervisorId]);
-      }
```

Comment:
Ok, yeah...this was new developer code. 😆

Reply:
I figured it was an intern. lol